errbot/core_plugins/chatRoom.py

```diff
@@ -235,6 +235,8 @@ def callback_message(self, msg):
             elif msg.is_group:
                 fr = msg.frm
                 chat_room = str(fr.room)
+                # Hipchat Server, this might be a JID, look it up just in case
+                chat_room = self.query_room(chat_room)
```

Comment:
It looks super hackish, what is really going on ? why fr.room is not a room from the hipchat backend ? (the bug is in the backend, not core I guess ?)